src/index.html

```diff
@@ -4,9 +4,305 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="Moyo-logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="content">
+        <!-- Card-block-1 -->
+        <div class="content__card card-block" data-qa="card">
+          <img
+            class="card-block__img"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card-block__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+          <p class="card-block__code">Код товара: 195434</p>
+          <div class="card-block__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card-block__comment">Отзывов: 5</span>
+          </div>
+          <div class="card-block__price">
+            <span class="card-block__price-text">Цена:</span>
+            <span class="card-block__price-num">69 999 грн</span>
+          </div>
+          <a
+            class="card-block__link"
+            href="#"
+            data-qa="card-hover"
+          >
+          Купить
+          </a>
```

Comment:
```suggestion
          >
            Купить
          </a>
```
indent before 'купить'.  And to other 7 cards

Reply:
fix indent